lessons/micropython/mini-workshop/index.md

```diff
@@ -0,0 +1,319 @@
+
+!!! note ""
+
+    Tento workshop nejde projít jen tak z domu – je potřeba speciální příprava.
+    Pokyny pro organizátory najdeš
+    na [podstránce]({{ subpage_url('organizers') }}).
+
+Jedna varianta Pythonu, MicroPython, běží na malých zařízeních.
+Taková zařízení si teď můžeš vyzkoušet.
+
+Jak na to?
+
+
+## Otevření příkazové řádky
+
+<img src="{{ static('icon_terminal.png')}}" style="float:right;height:5em;">
```

Comment:
I don't see much reason to prefer vectors here.

Also, the [source SVG in the Adwaita repo](https://github.com/GNOME/adwaita-icon-theme/blob/master/src/fullcolor/accessories-text-editor.svg) has extra stuff (and I don't want to edit it), and I could only find a PNG installed on my machine. So I used that one.